railties/test/application/rake/dbs_test.rb

```diff
@@ -545,15 +545,15 @@ def db_fixtures_load(expected_database)
 
         assert_equal '["posts"]', list_tables[]
         rails "db:schema:load"
-        assert_equal '["posts", "comments", "schema_migrations", "ar_internal_metadata"]', list_tables[]
+        assert_equal ["ar_internal_metadata", "comments", "posts", "schema_migrations"], JSON.parse(list_tables[]).sort
```

Comment:
Why do we need to parse json and sort this?

Reply:
If we need to sort for some reason we can do in the `runner` call.

Reply:
Those tests failed because `connection.tables` no longer is in order of creation. I thought it would be cleanest to change the tests to only assert on array content. I'll sort in the `runner` call instead.

Alternatively, I could join on `pragma_table_list` instead of selecting it directly, and rely on `sqlite_master.rowid` for ordering. Please let me know if you would prefer that.